### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by hlomzik**

### CODE REVIEW

File: src/tags/object/Audio.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import { ErrorMessage } from "../../components/ErrorMessage/ErrorMessage";
 import { AnnotationMixin } from "../../mixins/AnnotationMixin";
 
 /**
- * Audio tag plays a simple audio file.
+ * Audio tag plays a simple audio file
```
Comment (smoreface) [inline src/tags/object/Audio.js:15]: does the period break something?

File: src/regions/AudioRegion.js [hunk 2]
```diff
@@ -50,6 +50,26 @@ const Model = types
...
+     * {
+     *   "original_length": 18,
+     *   "value": {
+     *     "start": 3.1,
+     *     "end": 8.2,
+     *     "labels": ["Voice"]
+     *   }
+     * }
+     * @typedef {Object} AudioRegionResult
+     * @property {number} original_length length of the original audio (s)
```
Comment (smoreface) [inline src/regions/AudioRegion.js:64]:
```suggestion
     * @property {number} original_length length of the original audio
```

File: src/regions/AudioRegion.js [hunk 3]
```diff
@@ -50,6 +50,26 @@ const Model = types
...
+     * @property {Object} value
+     * @property {number} value.start start time of the fragment (s)
```
Comment (smoreface) [inline src/regions/AudioRegion.js:66]:
```suggestion
     * @property {number} value.start start time of the fragment(s)
```

File: src/regions/AudioRegion.js [hunk 4]
```diff
@@ -50,6 +50,26 @@ const Model = types
...
+     * @property {number} value.end finish time of the fragment (s)
```
Comment (smoreface) [inline src/regions/AudioRegion.js:67]:
```suggestion
     * @property {number} value.end finish time of the fragment(s)
```

File: src/tags/object/Audio.js (hunk 1 above)
Comment (hlomzik) [reply src/tags/object/Audio.js:15]: of course no :)
we can add it back for sure

File: src/regions/AudioRegion.js [hunk 5]
```diff
@@ -61,10 +61,10 @@ const Model = types
      *   }
      * }
      * @typedef {Object} AudioRegionResult
-     * @property {number} original_length length of the original audio (s)
+     * @property {number} original_length length of the original audio
```
Comment (hlomzik) [inline src/regions/AudioRegion.js:64]: That's a format of the value :)
seconds in this case
I put it for further automatic data generation
I'll return it back, but if you have any idea, how to make it better, I'd be glad to hear!

Comment (hlomzik) [reply src/regions/AudioRegion.js:64]: I'll use `(seconds)`

Comment (smoreface) [reply src/regions/AudioRegion.js]: perfect, thank you! I didn't realize that's what it was lol

**Review by smoreface — APPROVED** [review]

**Pull request merged**